### PR TITLE
Improved German Translations (de.json)

### PR DESCRIPTION
**Pull request by josiebytes**
## Description

Improved German translations

## Type of change

- [ ] Translation updates (fix/improve or add new translations)

My Bootstrap Academy username: Joshiy13

### CODE REVIEW

File: locales/de.json [hunk 1]
```diff
@@ -130,7 +130,7 @@
     "Challenge": "Challenge",
     "Explanation": "Erklärung",
     "Input": "Eingabe",
-    "ProgramInputShouldBe": "Wenn die Eingabe in Ihren Code erfolgt",
+    "ProgramInputShouldBe": "Eingabe sollte sein",
     "ProgramOutputShouldBe": "Ausgabe sollte sein",
```
Comment (Defelo) [inline locales/de.json]:
```suggestion
    "ProgramInputShouldBe": "Eingabe:",
    "ProgramOutputShouldBe": "Erwartete Ausgabe:",
```

**Review by Defelo — CHANGES_REQUESTED** [review]

File: locales/de.json (hunk 1 above)
Comment (josiebytes) [reply locales/de.json]: I think "Erwartete Ausgabe:" is fine but for Consistency "Eingabe:" should be "Erwartete Eingabe:" right? Or am I understanding the context wrong?

Comment (Defelo) [reply locales/de.json]: Yeah, the problem is, that the translation key is actually wrong here, something like `ProgramInput` would be more appropriate. Of course you could also fix that, if you would like to.

Comment (josiebytes) [reply locales/de.json]: Okay I will fix that

Comment (josiebytes) [reply locales/de.json]: Fixed!

**Review by Defelo — APPROVED** [review]

**Pull request merged**